rebasehelper/checkers/abipkgdiff_tool.py

```diff
@@ -112,17 +119,97 @@ def run_check(cls, results_dir, **kwargs):
             command.append(pkg)
             command.append(new_pkg)
             logger.debug('Package name for ABI comparison %s', old_name)
-            output = os.path.join(cls.results_dir, results_dir, old_name + '-' + cls.log_name)
+            output = os.path.join(cls.results_dir, old_name + '.txt')
             try:
                 ret_code = ProcessHelper.run_subprocess(command, output_file=output)
             except OSError:
                 raise CheckerNotFoundError("Checker '%s' was not found or installed." % cls.CMD)
 
             if int(ret_code) & settings.ABIDIFF_ERROR and int(ret_code) & settings.ABIDIFF_USAGE_ERROR:
                 raise RebaseHelperError('Execution of %s failed.\nCommand line is: %s' % (cls.CMD, cmd))
-            if int(ret_code) == 0:
-                text = 'ABI of the compared binaries in package %s are equal.' % old_name
-            else:
-                text = 'ABI of the compared binaries in package %s are not equal.' % old_name
-            reports[output] = text
-        return reports
+            reports[old_name] = int(ret_code)
+        return {'packages': cls.parse_abi_logs(reports), 'abi_changes': cls.abi_changes, 'path': cls.results_dir}
+
+    @classmethod
+    def parse_abi_logs(cls, reports):
+        """
+        Parse abipkgdiff logs
+        :param reports: dictionary with paths to the logs of the produced (sub)packages
+        :return: returns packages names with its abipkgdiff changes
+        """
+        def parse_changes(lines):
+            """
+            Parses each line with abipkgdiff output info
+            :param lines: lines of changes
+            :return: dictionary of list of changes
+
+            Example abipkgdiff line:
+            '  Functions changes summary: 3 Removed, 0 Changed, 0 Added functions (4 filtered out)'
+            """
+            summary_re = re.compile(r'''^
+            \s+(?P<kind>[\w\s]+changes\s+summary):\s+
+            (?P<changes>.+)
+            $
+            ''', re.VERBOSE)
+            changes_re = re.compile(r'''
+            (?P<count>\d+)\s+
+            (?P<what>Added|Changed|Removed)(\s+functions)?
```

Comment:
I think the last group should look like this: `(\s+functions|variables)?`. Otherwise the regexp won't match `Variables changes summary: ...`.